pages/Meat-Jerky.md

```diff
@@ -0,0 +1,5 @@
+*"Extra saturating"*
```

Comment:
There is no need to repeat the lore on here, it doesn't really give much information about the Item.